src/components/views/settings/tabs/user/KeyboardUserSettingsTab.tsx

```diff
@@ -76,6 +77,12 @@ interface IKeyboardShortcutRowProps {
     name: string;
 }
 
+// Filter out the labs section if labs aren't enabled.
+const visibleCategories = SdkConfig.get()['showLabsSettings']
+    ? Object.entries(CATEGORIES)
+    : Object.entries(CATEGORIES)
+        .filter(([categoryName]) => categoryName !== CategoryName.LABS);
```

Comment:
I think I'd prefer something like `Object.entries(CATEGORIES).filter([categoryName] => categoryName !== CategoryName.LABS || SdkConfig.get()['showLabsSettings'])` but that is probably just a personal preference